### PR TITLE
[Inference] Add models CodeLlama-7b and falcon-7b

### PR DESCRIPTION
**Pull request by Deegue**

### CODE REVIEW

**Comment by Deegue** [comment]
Gentle ping @jiafuzha for review, thanks~

**Comment by jiafuzha** [comment]
you need to add the two models in the include list so that they can be verified in PR. Otherwise, they will only be verified in nightly.

include:
          - { model: "gpt-j-6b"}
          - { model: "mistral-7b-v0.1"}
          - { model: "mpt-7b-bigdl"}
          - { model: <your models>
          - dtuner_model: nathan0/mpt-7b-deltatuner-model
            model: mpt-7b

**Comment by Deegue** [comment]
> you need to add the two models in the include list so that they can be verified in PR. Otherwise, they will only be verified in nightly.
> 
> include: - { model: "gpt-j-6b"} - { model: "mistral-7b-v0.1"} - { model: "mpt-7b-bigdl"} - { model: - dtuner_model: nathan0/mpt-7b-deltatuner-model model: mpt-7b

Ic.. it passed in our previous repo. Let me trigger them here.

**Comment by Deegue** [comment]
Gentle ping @jiafuzha for another review, thanks!

File: .github/workflows/workflow_inference.yml [hunk 1]
```diff
@@ -45,6 +45,8 @@ jobs:
           - { model: "gpt-j-6b"}
           - { model: "mistral-7b-v0.1"}
           - { model: "mpt-7b-bigdl"}
+          - { model: "CodeLlama-7b-hf"}
+          - { model: "falcon-7b"}
```
Comment (jiafuzha) [inline .github/workflows/workflow_inference.yml:49]: please review them before merging it since we've verified them successfully.

Comment (Deegue) [reply .github/workflows/workflow_inference.yml:49]: Ok, I will remove it as soon as it's ready to merge.

Comment (Deegue) [reply .github/workflows/workflow_inference.yml]: Removed.

**Review by jiafuzha — APPROVED** [review]
LGTM

**Pull request merged**